### PR TITLE
update support new models doc

### PR DESCRIPTION
**Pull request by yichaolemon**
## Checklist

- [ ] Format your code according to the [Format code with pre-commit](https://docs.sglang.ai/developer_guide/contribution_guide.html#format-code-with-pre-commit).
- [ ] Add unit tests according to the [Run and add unit tests](https://docs.sglang.ai/developer_guide/contribution_guide.html#run-and-add-unit-tests).
- [ ] Update documentation according to [Write documentations](https://docs.sglang.ai/developer_guide/contribution_guide.html#write-documentations).
- [ ] Provide accuracy and speed benchmark results according to [Test the accuracy](https://docs.sglang.ai/developer_guide/contribution_guide.html#test-the-accuracy) and [Benchmark the speed](https://docs.sglang.ai/developer_guide/contribution_guide.html#benchmark-the-speed).

### CODE REVIEW

File: docs/supported_models/support_new_models.md [hunk 1]
```diff
@@ -65,14 +67,22 @@ should give the same text output and very similar prefill logits:
...
+
 
-This is the command to test a new model on your local machine:
+This is an example command to run to test a new model on your local machine:
 
 ```bash
 ONLY_RUN=Qwen/Qwen2-1.5B python3 -m unittest test_generation_models.TestGenerationModels.test_others
 ```
 
+### Benchmark
```
Comment (XiaotongJiang) [inline docs/supported_models/support_new_models.md:80]: we should note the minimum requirement is MMMU as benchmark for PR merge, but other eval is encouraged too to be included in the PR description

Comment (yichaolemon) [reply docs/supported_models/support_new_models.md:80]: done!

File: docs/supported_models/support_new_models.md [hunk 2]
```diff
@@ -65,14 +67,21 @@ should give the same text output and very similar prefill logits:
 To ensure the new model is well maintained, add it to the test suite by including it in the `ALL_OTHER_MODELS` list in
 the [test_generation_models.py](https://github.com/sgl-project/sglang/blob/main/test/srt/models/test_generation_models.py)
 file, test the new model on your local machine and report the results on demonstrative benchmarks (GSM8K, MMLU, MMMU,
-MMMU-Pro, etc.) in your PR.
+MMMU-Pro, etc.) in your PR. \\
+For VLMs, also include a test in [test_vision_openai_server_a.py](https://github.com/sgl-project/sglang/blob/main/test/srt/test_vision_openai_server_a.py) or [test_vision_openai_server_b.py](https://github.com/sgl-project/sglang/blob/main/test/srt/test_vision_openai_server_b.py).
```
Comment (yichaolemon) [inline docs/supported_models/support_new_models.md]: @mickqian lmk if this makes sense -- not sure about the difference between these 2 files

Comment (JustinTong0323) [reply docs/supported_models/support_new_models.md]: just split for CI, avoid one long test

**Comment by JustinTong0323** [comment]
Good job!
May you also help update here:
```
2. **Register a new chat-template**
   See [conversation.py](https://github.com/sgl-project/sglang/blob/86a779dbe9e815c02f71ea82574608f6eae016b5/python/sglang/srt/conversation.py)
```
At present, we support the directly applying given chat templates. If the provided chat template is capable of handling images, there is no necessity to register a new one here.

**Comment by yichaolemon** [comment]
> Good job! May you also help update here:
> 
> ```
> 2. **Register a new chat-template**
>    See [conversation.py](https://github.com/sgl-project/sglang/blob/86a779dbe9e815c02f71ea82574608f6eae016b5/python/sglang/srt/conversation.py)
> ```
> 
> At present, we support the directly applying given chat templates. If the provided chat template is capable of handling images, there is no necessity to register a new one here.

Updated, thanks!

File: docs/supported_models/support_new_models.md (hunk 2 above)
Comment (mickqian) [reply docs/supported_models/support_new_models.md]: You can replace with `test_vision_openai_server_x.py`

Comment (yichaolemon) [reply docs/supported_models/support_new_models.md]: done

**Review by JustinTong0323 — APPROVED** [review]

File: docs/supported_models/support_new_models.md [hunk 3]
```diff
@@ -65,14 +67,21 @@ should give the same text output and very similar prefill logits:
...
-This is the command to test a new model on your local machine:
+This is an example command to run to test a new model on your local machine:
 
 ```bash
 ONLY_RUN=Qwen/Qwen2-1.5B python3 -m unittest test_generation_models.TestGenerationModels.test_others
 ```
 
+### Benchmark
+
+- **(Required) MMMU**: follow MMMU benchmark [README.md](https://github.com/sgl-project/sglang/blob/main/benchmark/mmmu/README.md) to get SGLang vs. HF Transformer accuracy comparison. The accuracy score from SGLang run should not be much lower than that from HF Transformer run. Similarly, follow https://docs.sglang.ai/developer_guide/benchmark_and_profiling.html to get performance comparison: TTFT and throughput must meet or exceed baselines (e.g., HF Transformer).
```
Comment (mickqian) [inline docs/supported_models/support_new_models.md:82]: Hi, please move this to a section/doc file for VLM specifically
